batch/src/main/scala/uk/gov/ons/addressindex/models/HybridAddressNisraEsDocument.scala

```diff
@@ -226,7 +235,8 @@ object HybridAddressNisraEsDocument extends EsDocument {
       "altThoroughfare" -> splitAndCapitalise(Option(row.getString(17)).getOrElse("")),
       "dependentThoroughfare" -> splitAndCapitalise(Option(row.getString(18)).getOrElse("")),
       "locality" -> splitAndCapitalise(Option(row.getString(19)).getOrElse("")),
-      "townland" -> splitAndCapitalise(Option(row.getString(20)).getOrElse("")),
+   //   "townland" -> splitAndCapitalise(Option(row.getString(20)).getOrElse("")),
+      "udprn" -> (if (row.isNullAt(20) || row.getString(20).equals("")) null else toInt(row.getString(20)).orNull),
```

Comment:
`toInt` returns `None` if the string is empty. These statements can be simplified as `toInt(row.getString(20)).orNull`